### ISSUE TITLE
A Path from Logic to Metaphysics

### ISSUE THREAD

**Issue by dwbcampbell** [michaelt/martin-lof#11]
... seems to be missing pages 3 & 7 and have two page 4s.

**Comment by michaelt** [comment]
The catalogue for the university library is telling me it has this volume, I will see if I can scan it.

**Comment by jimmyhmiller** [comment]
@michaelt  Any news on whether you have it? I'd love to read the article, but it is hard to follow with the missing pages.

**Comment by michaelt** [comment]
I finally got the book, here is one way of accessing the (published) text. http://archive-pml.github.io/martin-lof/pdfs/A-path-from-logic-to-metaphysics-1991.pdf  I decided to just add it as the 'published' version.

As far as I can tell almost all bibliographical references to this work are wrong. I should have had the sense to add the title page when scanning it.  The following is from my library's database:

------------------------

Title:	

Atti del Congresso Nuovi problemi della logica e della filosofia della scienza / organizzato dalla Società italiana di logica e filosofia delle scienze (SILFS), Viareggio, 8-13 gennaio 1990 ; a cura di Domenico Costantini e Maria Carla Galavotti. 

Author:	

Congresso Nuovi problemi della logica e della filosofia della scienza (1990 : Viareggio, Italy)


Publisher:	Bologna : Editrice CLUEB,  copyright 1991.

------------------

In fact on the title page, the bit "a cura di Domenico Costantini e Maria Carla Galavotti" does not appear - this is presumably correct for volume one. Here we have instead

Volume II - Logica; a cura di Giovanna Corsi e Giovanni Sambin
